.prettierrc.yml

```diff
@@ -0,0 +1,7 @@
+semi: true
+trailingComma: 'all'
```

Comment:
Also not a huge fan of this.  Just seems odd to throw commas after function call arguments.  But I could probably live with this if I must!

Reply:
Does it help that you don't actually have to type the commas yourself?

Reply:
Yeah, it's fine.  Just fighting years of training my brain to track down trailing commas in javascript errors in IE.